oak-run/src/main/java/org/apache/jackrabbit/oak/run/RevisionsCommand.java

```diff
@@ -118,7 +125,7 @@ public RevisionsCommand(boolean exitWhenDone) {
         this.exitWhenDone = exitWhenDone;
     }
 
-    private static class RevisionsOptions extends Utils.NodeStoreOptions {
+    public static class RevisionsOptions extends Utils.NodeStoreOptions {
```

Comment:
```suggestion
    static class RevisionsOptions extends Utils.NodeStoreOptions {
```

So with above changes we can reduce this to package-only.

Reply:
Done, thank you for the suggestion.